src/resolvers/projectVerificationFormResolver.ts

```diff
@@ -183,19 +184,10 @@ export class ProjectVerificationFormResolver {
       await projectVerificationForm.save();
 
       const callbackUrl = `https://${dappUrl}/verification/${project.slug}/${token}`;
```

Comment:
if the callback url is not used, lets remove it here. I see you build it inside the sendMailConfirmationEmail method.